### DISCUSSION TITLE
Supplies

### DISCUSSION THREAD

**Discussion by DiscoSlug** [hargata/lubelog#307] [Ideas]
I had two potential ideas for handling used supplies. I see that they remain and are not hidden and I'm sure there is a reason for that. Currently they are reading zero quantity, so would it be possible to push "0" quantity supplies to the bottom of the page? I have it sorting by date currently with newest at the top, which is what would be preferred for all other pages and this one, but if it could override 0 that would be great. 

Or, even better but I'm sure would be more trouble, having a "used" supplies or hidden area where used supplies are moved to. 

If not, no worries! It works great as is, and the --- & "0" is enough to let me know I've used that supply already.

**Comment by hargata** [comment]
@DiscoSlug hey will this more or less fulfill this requirement? https://youtu.be/J2ymGhFuTTw?si=08fOCYPtY3W7MwQd&t=106

**Comment by DiscoSlug** [comment]
This is a great feature! 

I think this new feature would work for me with just one more change. I see in the video (this and the last one describing use of supplies) that you keep going back to the supply to add more in when your plan runs out. 

Could there be a positive inventory adjustment just like the negative ones pulled by the service record? So I have 3L left of oil but my truck needs 6L. I go buy a 10L jug. That 10L was $60, if I add the jug as a new supply (maybe it is different brand), it will not be pulled by the plan. But, if I had a supply of "10W30" oil, with inventory additions from supply purchases that are added to it, then that would work? 

This is probably more complicated though.

**Reply by hargata** [reply]
Yeah...the next thing I was going to work on was to make replenishing supplies a lot easier, I know from other users they tend to just have one record per supply(e.g.: 10w30 motor oil) and then they just add to the quantity / total when they run out or when they replenish it. The idea is that in next release, there will be a feature that will let you replenish the supplies, but that's still being designed.

**Reply by DiscoSlug** [reply]
I'll be watching! Thanks for the great program.

**Reply by hargata** [reply]
This is now released as of 1.2.2, a new mechanism of replenishing supply: https://youtu.be/m_FAbZODdyY?si=Y_x2H30q6bF4kvQs&t=389
